src/main/java/io/lumigo/core/utils/AwsUtils.java

```diff
@@ -282,10 +308,44 @@ private static String extractMessageIdFromDynamodbRecord(
             DynamodbEvent.DynamodbStreamRecord record) {
         if (record.getEventName() == null) return null;
         if (record.getEventName().equals("INSERT")) {
-            return dynamodbItemToHash(record.getDynamodb().getNewImage());
+            return calculateItemHash(record.getDynamodb().getNewImage());
```

Comment:
what is the purpose of these changes?

Reply:
when i updated the lambda event library this got broken because in the new version they created the same object but inside this library (before they import it from dynamodb package)
so now the old logic isnt working becuase its different classes (with the same name, but methods with different names) so i needed to implement it again